components/infieldbutton/stories/infieldbutton.stories.js

```diff
@@ -1,80 +1,85 @@
 import { default as IconStories } from "@spectrum-css/icon/stories/icon.stories.js";
+import { Sizes, withDownStateDimensionCapture } from "@spectrum-css/preview/decorators";
 import { disableDefaultModes } from "@spectrum-css/preview/modes";
-import { isActive, isDisabled, isFocused, isHovered, isQuiet, size } from "@spectrum-css/preview/types";
+import { isActive, isDisabled, isHovered, isQuiet, size } from "@spectrum-css/preview/types";
 import metadata from "../dist/metadata.json";
 import packageJson from "../package.json";
 import { InfieldButtonGroup } from "./infieldbutton.test.js";
-import { Template } from "./template.js";
+import { InfieldButtonGroupVariant, InfieldButtonIcons, Template } from "./template.js";
 
 /**
- * The in-field button component is a button used inside a text field.
+ * In-field buttons are used to represent actions within input fields. They’re currently used inside the [number field,](/docs/components-stepper--docs).
  */
 export default {
 	title: "In-field button",
 	component: "InFieldButton",
 	argTypes: {
 		size: size(["s", "m", "l", "xl"]),
 		isQuiet,
-		position: {
-			name: "Position",
-			type: { name: "string", required: true },
-			table: {
-				type: { summary: "string" },
-				category: "Component",
-			},
-			options: ["left", "right", "top", "bottom"],
-			control: "select"
-		},
 		iconName: {
-			...IconStories?.argTypes?.iconName ?? {},
-			if: false,
+			...IconStories?.argTypes?.uiIconName ?? {},
+			options: ["ChevronDown", "Cross", "Dash", "Add"],
+			if: { arg: "isInline", neq: true },
+			description: "These are icons to use within the in-field button - `ChevronDown`, `Add`, `Dash` and `Cross`",
 		},
 		isDisabled,
-		isFocused,
 		isActive,
 		isHovered,
-		isStacked: { table: { disable: true } },
+		isInline: {
+			name: "Side by side",
+			type: { name: "boolean" },
```

Comment:
I'll leave this up to you, but some of our other components have control descriptions, so we could add a `description` here too. It could just be simple, like "Renders two infield buttons to be used as a numeric stepper" or something even more general. 

Just a thought, not a blocker!